draft-ietf-quic-transport.md

```diff
@@ -5018,6 +5034,31 @@ sequence number, or if a sequence number is used for different connection
 IDs, the endpoint MAY treat that receipt as a connection error of type
 PROTOCOL_VIOLATION.
 
+The Retire Prior To field is a requirement for the peer to retire all connection
+IDs with a sequence number less than the specified value.  This includes the
+initial and preferred_address transport parameter connection IDs.  The peer
+SHOULD immediately retire all the connection IDs.
+
+The sender of the NEW_CONNECTION_ID frame MAY remove the connection IDs after 3
+PTO, even if the peer has not retired them yet.  The 3 PTO timer starts on
+acknowledgement of the packet containing the NEW_CONNECTION_ID frame.  Continued
+use of the retired connection IDs after this point will likely result in a
+stateless reset being sent.
+
+In order to prevent a stateless reset, sent in response to a heavily delayed
+packet using a retired connection ID, from closing the connection, the
```

Comment:
```suggestion
packet using a retired connection ID from closing the connection, the
```
This parenthetical phrase is pretty critical and this sentence is pretty comma-heavy.  Consider promoting it.